django/contrib/admin/utils.py

```diff
@@ -340,12 +340,9 @@ def label_for_field(name, model, model_admin=None, return_attr=False):
             # field is likely a ForeignObjectRel
             label = field.related_model._meta.verbose_name
     except FieldDoesNotExist:
-        if name == "__unicode__":
+        if name == "__str__":
```

Comment:
Is there a behavior changed here since `attr = bytes` changed to `attr = str`? I'm not sure what the consequences are.